src/zkey/pkey.c

```diff
@@ -123,7 +167,39 @@ bool is_ep11_aes_key(const u8 *key, size_t key_size)
 	if (ep11key->head.length > key_size)
 		return false;
 
-	if (ep11key->version != 0x1234)
+	if (ep11key->version != EP11_STRUCT_MAGIC)
+		return false;
+
+	return true;
+}
+
+/**
+ * Check if the specified key is an EP11 ECC key token with header
+ * (TOKVER_EP11_ECC_WITH_HEADER). This means we have a 16-byte ep11kblob_header
+ * followed by a ep11keytoken struct with an own 32-byte header (session id).
```

Comment:
the session Id is NOT a header, ist simply part of the EP11 key blob. So I would not state it that way....

Reply:
Done.